### PR TITLE
[Snapcraft] - Added snapcraft last update badge 

### PR DESCRIPTION
**Pull request by MohanKumarAmbati**
- Added last updated badge for Snapcraft. 

Hi @chris48s, can you please add "hacktoberfest-accepted" label to this PR so that it is considered under it.

closes - #10300

Thanks in advance.

### CODE REVIEW

**Review by chris48s — DISMISSED** [review]
This looks good. No notes from me :+1:

I just noticed that there was already a review app deployed for this, so I'm going to leave it open for the moment as I suspect @pyvesb might also be having a look.

**Review by PyvesB — COMMENTED** [review]
Yes, I was having a look as well. One small casing correction, otherwise looks good to me :)

**Comment by PyvesB** [comment]
> can you please add "hacktoberfest-accepted" label to this PR so that it is considered under it.

I've opted the whole repository into Hacktoberfest, this should no longer be needed.

File: services/snapcraft/snapcraft-last-update.service.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          snapcraftPackageParam,
+          ...pathParams(
+            { name: 'track', example: 'latest' },
+            { name: 'risk', example: 'stable' },
+          ),
+          queryParam({
+            name: 'arch',
+            example: 'amd64',
+            description:
+              'Architecture, When not specified, this will default to `amd64`.',
```
[the remark this answers is not in the document]
Comment (MohanKumarAmbati) [reply services/snapcraft/snapcraft-last-update.service.js]: made changes accordingly.

**Comment by MohanKumarAmbati** [comment]
> > can you please add "hacktoberfest-accepted" label to this PR so that it is considered under it.
> 
> I've opted the whole repository into Hacktoberfest, this should no longer be needed.

Thank you @PyvesB

**Review by PyvesB — APPROVED** [review]

**Pull request merged**